spec/features/customer_orders/new_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'rails_helper'
+
+RSpec.describe 'Customer Orders Index' do
+  it 'shows a link to create a new order' do
+    customer = Customer.create!({
+      first_name: 'Chaz',
+      last_name: 'Simons',
+      address: '123 Cool Guy Street',
+      over_21: true,
+      rewards: 1000
+      })
+    visit "/customers/#{customer.id}/orders"
+    click_link 'Create Order'
```

Comment:
i'm not positive about this, but I don't think `click_link` is an assertion. you should add in a
`expect(current_path).to eq "whatever the path is"`